lib/puppeteer-helpers.js

```diff
@@ -122,6 +122,16 @@ function getAttribute ($element, attributeName) {
   return $element.evaluate((el, name) => el.getAttribute(name), attributeName)
 }
 
+/**
+ * Gets the accessible name of the given element, if it exists in the accessibility tree
+ *
+ * @param {import('puppeteer').ElementHandle} $element - Puppeteer element handle
+ * @returns {string} - The element's accessible name
+ */
+async function getAccessibleName ($element) {
+  return (await page.accessibility.snapshot({ root: $element })).name
```

Comment:
Because of a potential `Promise<null>` return we'll need a quick `?.` I guess?

```suggestion
  return (await page.accessibility.snapshot({ root: $element }))?.name
```